lntest/harness.go

```diff
@@ -648,6 +649,57 @@ func (n *NetworkHarness) StopNode(node *HarnessNode) error {
 	return node.stop()
 }
 
+// SaveProfilesPages hits profiles pages of all active nodes and writes it to
+// disk using a similar naming scheme as to the regular set of logs.
+func (n *NetworkHarness) SaveProfilesPages() {
+	for _, node := range n.activeNodes {
+		resp, err := http.Get(
```

Comment:
not blocking, but might help with indentation to define a `saveProfilePage` helper

Reply:
Done